src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -180,8 +180,14 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 },
 
                 get: el => LocationAccessorsInstrumentation.isLocationWrapper(el) ? el.href : PropertyAccessorsInstrumentation._getUrlAttr(el, 'href'),
-                set: (el, value) => LocationAccessorsInstrumentation.isLocationWrapper(el) ?
-                                    el.href = destLocation.resolveUrl(value, document) : el.setAttribute('href', value)
+                set: (el, value) => {
+                    if (LocationAccessorsInstrumentation.isLocationWrapper(el))
+                        el.href = destLocation.resolveUrl(value, document);
+                    else
+                        el.setAttribute('href', value);
+
+                    return void 0;
```

Comment:
BTW FYI @churkin @miherlosev @LavrovArtem WHATWG just standartized location object: https://html.spec.whatwg.org/multipage/browsers.html#the-location-interface Might be useful